src/Runtime/platform.h

```diff
@@ -151,4 +151,8 @@ typedef void (*Event)(void);
 #define U32_MAX 0xFFFFFFFF
 #define F32_MAX 3.4028235E38
 
+#define FLAGS_NONE 0
+#define HAS_FLAG(value, flag) (((value) & (flag)) != FLAGS_NONE)
+#define NOT_FLAG(value, flag) (((value) & (flag)) == FLAGS_NONE)
```

Comment:
```suggestion
#define HAS_FLAG(value, flag) ((value) & (flag))
#define NOT_FLAG(value, flag) (!((value) & (flag)))
```

Reply:
Kind of the same thing as `!gobj` vs `!= NULL` and wouldn't work in a language like C# but if you think it's better then ok.